vars/Debian.yml

```diff
@@ -1 +1,6 @@
 ---
+
+icinga2_master_ido_packages:
+  - "python-mysqldb"
+  - "icinga2-ido-mysql"
+  - "mysql-client"
```

Comment:
I'm not sure if Ansible still complains about that but I think we should define `icinga2_master_ca_path` here as well.